### DISCUSSION TITLE
Possible error

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2227] [Translations]
Dear @kajaji 

in Arabic sky cultures I see phrase "Stoopness [Of The Camel Neck]" - maybe this is a "Steepness [Of The Camel Neck]"?

**Comment by kajaji** [comment]
Yes, I thought stoopness is a noun from "stoop" (bend downward), but "stoopness" is not found as an English word.

We can use the phrase "The downward Bend".

**Reply by alex-w** [reply]
Would you prepare patch?

**Reply by kajaji** [reply]
I tried to create a patch but it came out empty. I squashed all the commits in my fork here:
[https://github.com/kajaji/stellarium/tree/master](https://github.com/kajaji/stellarium/tree/master)

Shall I make a pull request

**Reply by alex-w** [reply]
Probably “The Downward Bend [of the Camel Neck]”?

**Reply by kajaji** [reply]
I prefer to leave out "[of the Camel Neck]". After examining again Ref. no.1 for Arabic (indigenous) sc, and the Arab Lexicon "Taj al-aroos", it appears the name has to do with a thing that is bent, in general, and not to the camel neck in particular.

**Reply by alex-w** [reply]
“The Downward Bend [of the Neck]” - to avoid misunderstanding?

**Reply by kajaji** [reply]
Thanks Alex for making me revisit this name.
In Ref. 2 and in Ref.3 of the Arabic (indigenous) skyculture, the  authors (both lived in the 10th century) state:
These two stars are named as such from the meaning of bending a thing unto itself, that each one is bent closer to the other.
Ref. 1 adds that these two stars rise after the three stars λ ,φ1, φ2 Ori, and because they are lower, they are named as such.

There is another meaning of the word, which is the brand (mark) on the lowe part of the camel neck. This meaning has been adopted by the late orientalist Prof. Paul Kunitzsch, but the references clearly did not associate the name to the camel or to the neck of any animal or bird with a long bent neck.

If the name is associated to a neck, it could be confusing with the image that these two stars are part of. They are a part of 8 stars depicting the bow of al-Jawza. (The drawing of al-Jawza and the bow is by Mr. Roland Laffitte).
![al-Hanaa](https://user-images.githubusercontent.com/34771982/152360005-bf509b03-4d9e-432c-9b5c-394bed77594e.png)

**Reply by alex-w** [reply]
Thank you very much for clarification! So, just "The Downward Bend", without any additions.
